apps/dashboard/pages/donations.tsx

```diff
@@ -7,13 +7,21 @@ import { useRouter } from 'next/router'
 import { useStrapiRequest } from '@wsvvrijheid/services'
 import { ssrTranslations } from '@wsvvrijheid/services/ssrTranslations'
 import { Donation, Sort, StrapiLocale } from '@wsvvrijheid/types'
+import { MonthPicker } from '@wsvvrijheid/ui'
 import { AdminLayout, DataTable, PageHeader, useColumns } from '@wsvvrijheid/ui'
 
 type PageProps = InferGetStaticPropsType<typeof getStaticProps>
+type Month = {
+  startYear: number
+  startMonth: number
+  endYear: number
+  endMonth: number
+}
 
 const DonationsPage: FC<PageProps> = ({ seo }) => {
   const [currentPage, setCurrentPage] = useState<number>(1)
   const [searchTerm, setSearchTerm] = useState<string>()
+  const [month, setMonth] = useState<Month>()
```

Comment:
`month.startYear` seems weird. State name could be something like `date`, `range` or `dateRange`

```suggestion
  const [date, setDate] = useState<Month>()
```